decidim-initiatives/lib/gem_overrides/origami/date.rb

```diff
@@ -0,0 +1,47 @@
+# frozen_string_literal: false
+
+module Origami
```

Comment:
I am a bit worried about the Origami gem, it seems to be stale at the moment.

There are also other issues with it:
https://github.com/gdelugre/origami/pull/74

They mention there that the maintainer has disappeared...

Reply:
> I am a bit worried about the Origami gem, it seems to be stale at the moment. 

I agree, the latest commits are from 2019. This was introduced in #4805, it's only used for the Initiative's PDF signature feature.  There are some alternatives described in #4753.

@arnaumonty @carolromero  is this feature actively used in Production? Or maybe we can rethink this feature, so we can find an alternative flow?

Reply:
There's also https://github.com/gettalong/hexapdf which seems to be able to "securing PDF files by encrypting or signing them", according to the Readme!

Reply:
Oh great, thanks! we'll take that into account, for sure seems like a much better alternative than the other options

Reply:
This is a stale repo, and yes, it makes sense to look into other solutions. At the moment, i do not think this is in the scope of Rails 3.0 upgrade.

Reply:
We discussed about this with @andreslucena and we concluded that we'll accept this monkey patch not to block the Ruby 3.0 upgrade and then later we'll refactor the code to another gem in another PR.

So I'll resolve this conversation for now.